src/components/pages/AddStaff/AddStaff.js

```diff
@@ -51,6 +51,14 @@ function AddStaff() {
       }));
     }
 
+    if(!/^[a-z0-9]+$/.test(state.username)){
+      return updateState(prevState => ({
+        ...prevState,
+        createAccount: false,
+        errors: [{username: 'Non-alphanumeric characters are not allowed'}]
```

Comment:
Errors to my understanding are stores in objects and not in arrays in this component

Reply:
Errors should be an array.
Reference:
```
  {state.errors.length && (
              <Message error>
                <Message.List>
                  {state.errors.map(error => (
                    <div key={uuid()}>
                      <Message.Item>{`${Object.values(error)}`}</Message.Item>
                    </div>
                  ))}
```
@quansenB